haiku-apps/fairtrade/fairtrade-1.0.0.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="The fair trade application supports sellers during the trade"
+DESCRIPTION="This application is designed for our local fair trade group (also see http://www.transfair.org/ (German link) for more info about the fair trade philosophy). The fair trade application supports sellers during the trade. Products can be scanned using a bar code scanner. All scanned products are showed in a shopping basket. For example, the seller scans a customer's product and the product appears in the shopping basket. If a customer wants to buy multiple products of the same kind, the product only has to be scanned once and the amount of products can be entered using the keypad. /
+Furthermore, it helps managing products on stock and provides an easy way to add new products. Sold products are automatically removed from the stock when completing a bargain. Product prices, descriptions, names and quantities can be edit as well. Old products can be disabled from the product view."
```

Comment:
DESCRIPTION should be wrapped to lines <= 80 chars (as you can see on most recipes.)
BTW, the current description is probably too long and needs some changes: "for our local fair group" shoud probablly be replaced by "for the faire trade group run by the owners of www.transfair.org.